packages/video-intelligence/README.md

```diff
@@ -0,0 +1,25 @@
+# NodeJS Client for Google Cloud Video Intelligence API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-node#versioning))
+
+Idiomatic NodeJS client for [Google Cloud Video Intelligence API][Product Documentation]
+- [Client Library Documentation][]
+- [Product Documentation][]
+
+## Quick Start
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+2. [Enable the video-intelligence api.](https://console.cloud.google.com/apis/api/video-intelligence)
+3. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/master/guides/authentication)
+
+### Installation
+```
+$ npm install @google-cloud/videointelligence
```

Comment:
Tracking this on the generator side here: https://github.com/googleapis/toolkit/issues/1290